edited data dictionary description (#3781)
catalyst-cooperative/pudl: docs/data_dictionaries/index.rst

Data Dictionaries
=================

Data Processed & Cleaned by PUDL
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> The PUDL data dictionary provides detailed information about the output tables,

Reviewer: ```suggestion
The PUDL data dictionary provides detailed metadata for core and output tables in the PUDL database,
```

including field names, data types, and descriptions

> for each field within the respective tables.

Reviewer: ```suggestion
for each field by table.
```

> for each field by table.

Reviewer 2: This wording might accidentally imply that there are other tables in the PUDL database that are not core or output tables that do not have metadata here. To avoid confusion, would say: 

"The PUDL data dictionary provides detailed metadata for the tables in the PUDL database. This includes table descriptions, field names, field, descriptions, and field datatypes."

.. toctree::
   :maxdepth: 1
   :titlesonly:

   pudl_db

Raw, Unprocessed Data
^^^^^^^^^^^^^^^^^^^^^

> This section outlines the raw, unprocessed data from the FERC forms,

Reviewer: ```suggestion
For data sources where substantial effort is required to publish raw databases (e.g., FERC Form 1), the dictionaries below provide key metadata on the raw data sources (e.g., table names and descriptions).
```
This section could grow to include metadata on other raw sources we're publishing, so maybe it's worth keeping it more general.

detailing the table names, links to their corresponding database tables,
raw file names, page numbers,

> data reporting frequency, and table descriptions.

Reviewer 2: I would go into a little more detail here and say something like: 

"Certain raw datasets (e.g. FERC Form 1) require additional effort to process. We load these raw sources into SQLite databases before feeding them into the PUDL data pipeline. The dictionaries below provide key metadata on these raw sources including table name, table description, links to corresponding database tables, raw file names, page numbers, and data reporting frequency.

Also @zaneselvans what exactly does the "Pages" column mean in these FERC1 table metadata descriptions? https://catalystcoop-pudl.readthedocs.io/en/nightly/data_dictionaries/ferc1_db.html

Data Dictionaries
=================

Data Processed & Cleaned by PUDL
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
The PUDL data dictionary provides detailed metadata for the tables
in the PUDL database. This includes table descriptions,

> field names, field, descriptions, and field datatypes.

Reviewer 2: Is the comma between field and descriptions is a typo?

Author: Yes, that was a mistake. It has been resolved

.. toctree::
   :maxdepth: 1
   :titlesonly:

   pudl_db

Raw, Unprocessed Data
^^^^^^^^^^^^^^^^^^^^^
Certain raw datasets (e.g. FERC Form 1) require additional effort to process.
We load these raw sources into SQLite databases before feeding them into
the PUDL data pipeline. The dictionaries below provide key metadata on
these raw sources including table name, table
description, links to corresponding database tables,
raw file names, page numbers, and data reporting frequency.

.. toctree::
   :maxdepth: 1
   :titlesonly:

   ferc1_db

Code Descriptions & Metadata
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
This section contains mappings of codes in the raw tables to their

> corresponding labels and descriptions in the processed PUDL database tables.

Reviewer 2: Might be a good idea to provide an example here.
